gravitee-rest-api-standalone/gravitee-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -170,6 +170,12 @@ services:
     enabled: true
     cron: "0 */5 * * * *"
 
+  # Subscription service
+  subscription:
+    enabled: true
+    #  Pre-expiration notification, number of days before the expiration an email should be send to subscriber and primary owner
+    pre-expiration-notification-schedule: 90,45,30
```

Comment:
to confirme with @ibahloul , but does this param need to be enabled by default ?

Reply:
yes